ee/billing/quota_limiting.py

```diff
@@ -310,7 +310,7 @@ def set_org_usage_summary(
             resource_usage["todays_usage"] = todays_usage.get(field, 0)
         else:
             # TRICKY: If we are not explictly setting todays_usage, we want to reset it to 0 IF the incoming new_usage is different
-            if (organization.usage or {}).get(field, {}).get("usage") != resource_usage.get("usage"):
+            if ((organization.usage or {}).get(field, {}) or {}).get("usage") != resource_usage.get("usage"):
```

Comment:
This is really ugly, but we can revert it once data warehouse pricing goes out and this value becomes properly filled in.

Reply:
Splititng the code up to something like this makes it more readable. It also make it easier to debug in Sentry because the var values will be in scope.

```
org_usage_data = organization.usage or {}
org_field_usage = org_usage_data.get(field, {})
org_usage = org_field_usage.get("usage")

if org_usage != resource_usage.get("usage"):
    # ...
```